Add Ruby/Basics exercise (#1043)
exercism/v3: languages/ruby/exercises/concept/basics/.docs/hints.md

### 1. Define the expected oven time in minutes

- You need to define a [constant][constant] which should contain the [integer][integers] value specified in the recipe.

### 2. Calculate the remaining oven time in minutes

> - You need to define a [method][methods] with a single parameter for the acutal time so far.

Reviewer: ```suggestion
- You need to define a [method][methods] with a single parameter for the actual time so far.
```

- You can [implicitly return an integer][return] from the method.
- You can use the [mathematical operator for subtraction][operators] to subtract values.

### 3. Calculate the preparation time in minutes

- You need to define a [method][methods] with a single parameter for the number of layers.
- You can [implicitly return an integer][return] from the method.
- You can use the [mathematical operator for multiplicaton][operators] to multiply values.

> - You could define an extra constant to help you, or use a "magic number" in the code.

Reviewer: Maybe make it explicit what to use the constant for?

```suggestion
- You could define an extra constant for the time in minutes per layer, or use a "magic number" in the code.
```
